docs/targets.md

```diff
@@ -0,0 +1,85 @@
+## Makefile Targets
+```
```

Comment:
This should be outputting the short help. Can you compare to the other modules to see why it's not working.